### PR TITLE
Template

### PR DESCRIPTION
**Pull request by 0xxlegolas**
- adding open source template

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-#*       @global-owner1 @global-owner2
+#*       @global-owner1 @global-owner2
```
Comment (neoyiu0x) [inline .github/CODEOWNERS:8]: do we need to specify the approver here for this open-source repo, i think we will also specify the roles for the codeowners for those project who might fork this.

File: .github/ISSUE_TEMPLATE/questions_and_others.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: "❓ Questions and Others"
+about: Questions regarding Klaytn DEX or other issues not related to bug nor feature request
```
Comment (neoyiu0x) [inline .github/ISSUE_TEMPLATE/questions_and_others.md]: as we have repo for backend and frontend, best to reflect it here as well thanks

File: .github/CODEOWNERS (hunk 1 above)
Comment (0xxlegolas) [reply .github/CODEOWNERS:8]: @neoofklaytn  I was also thinking on how it should be. Should we give all the emails contributed to this repo ?

File: .github/ISSUE_TEMPLATE/questions_and_others.md (hunk 2 above)
Comment (0xxlegolas) [reply .github/ISSUE_TEMPLATE/questions_and_others.md]: sure

File: .github/CODEOWNERS (hunk 1 above)
Comment (neoyiu0x) [reply .github/CODEOWNERS:8]: few from their side would be enough, and the majority should be from ours. 3 approvals required for every merge to happen, wdyt?

**Pull request merged**